README.md

```diff
@@ -45,6 +45,7 @@ Makes requests to `urls` and saves all files found with `sources` to `directory`
  - `urls`: array of urls to load and filenames for them *(required, see example below)*
  - `directory`: path to save loaded files *(required)*
  - `defaultFilename`: filename for index page *(optional, default: 'index.html')*
+ - `prettifyUrls`: Whether urls should be 'prettified', have the `defaultFilename` removed *(optional, default: false)*
```

Comment:
let's use lower case for first char in `Whether` in order to keep consistency with other options